### PR TITLE
Memory pair game release

### PR DESCRIPTION
**Pull request by mustbefail**
### Memory Pair Game

[Demo](https://mustbefail.github.io/memory-pair-game/) |
[Code base](https://github.com/mustbefail/memory-pair-game)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/mustbefail/memory-pair-game/app.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+const checkIdentity = () => {
+  const [first, second] = state.flippedCards.flippedCollection.map(el => el.dataset.cardId);
+  return first === second;
+};
+
+const CheckWinCondition = () => document.querySelectorAll('.hidden').length === 12;
+
+const flipCard = (card) => {
+  card.classList.add('flip');
```
Comment (AMashoshyna) [inline submissions/mustbefail/memory-pair-game/app.js:61]: I have two problems with this function:  1) it is called `flipCard` and receives one card as an argument, but under the hood is affects all cards (by calling updateCardState) 2) iw does two separate things - flip one card and update all the others, while ideally a function should do only one thing.

I would try to remove `updateCardsState()` call from this function and give this responsibility to some other entity

**Review by AMashoshyna — CHANGES_REQUESTED** [review]
Awesome job! Please see my comments below

**Comment by AMashoshyna** [comment]
Hey @mustbefail ! Do you need any assistance from my side with this task?

**Comment by mustbefail** [comment]
> Hey @mustbefail ! Do you need any assistance from my side with this task?

Hello. All clear. Postponed classes for a while. I got sick. I'm returning to duty.

**Comment by AMashoshyna** [comment]
Get well soon!

**Review by AMashoshyna — APPROVED** [review]

**Pull request merged**